lib/forty_facets/filter/scope_facet_filter_definition.rb

```diff
@@ -0,0 +1,45 @@
+module FortyFacets
+  class ScopeFacetFilterDefinition < FilterDefinition
+
+    attr_reader(:scopes)
+
+    def initialize search, scopes, opts
+      @search = search
+      @scopes = Array(scopes).flatten.map(&:to_sym)
+      @path = opts[:path]
+      @path ||= @scopes.join('-')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.